apps/zotonic_listen_smtp/src/zotonic_listen_smtp.erl

```diff
@@ -226,11 +227,11 @@ handle_RCPT(To, State) ->
         %     {error, "451 Server error. Please try again later.", State}
     end.
 
--spec handle_RCPT_extension(Extension :: binary(), State :: #state{}) -> {'ok', #state{}} | 'error'.
+-spec handle_RCPT_extension(Extension :: binary(), State :: #state{}) -> {'ok', #state{}}.
 handle_RCPT_extension(_Extension, State) ->
     {ok, State}.
```

Comment:
This function doesn't seem to be used anywhere nor to be doing much 😅 

Perhaps worth removing it now?

Reply:
It is a callback for the SMTP library.